debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -257,6 +270,39 @@ protected Iterable<Field> getAllConfigurationFields() {
         return OracleConnectorConfig.ALL_FIELDS;
     }
 
+    @Override
+    public void performCommit() {
+        if (!"xstream".equals(adapterType)) {
```

Comment:
Would not it be better to expose `io.debezium.connector.oracle.OracleConnectorConfig.connectorAdapter` and do enum based equality?